packages/lab/src/combo-box-next/ComboBoxNext.tsx

```diff
@@ -97,7 +97,10 @@ export interface ComboBoxNextProps<T>
    */
   itemFilter?: (source: T[], filterValue?: string) => T[];
   /* Callback for change event in input. */
-  onChange?: (event: SyntheticEvent, data: { value: string }) => void;
+  onChange?: (
```

Comment:
when I was playing around, I wonder why we split it into data: {value}, rather than just value, seems a bit unnecesssary to have to do that data.value all the time
but then also if its just going to be the event.target.value you can get from the event anyway, can we keep it simpler and just have onChange: (event) => void?

Reply:
Given the keyboard and click events are different, the value is in a different place in the event, so extracting the value saves you from having to check outside if it was from the keyboard or mouse to check for target or currentTarget.
about the data object, i would imagine it is to standardise outputs of handlers for cases in which we might send more than one parameter... idk if we do it anywhere else, i guess we could just send `value` but might be a good one for today's discussion